Setup/UpgradeSchema.php

```diff
@@ -0,0 +1,39 @@
+<?php
```

Comment:
@NandhiniKarthi I don't understand why we need this with declarative schema in place?